ml_datasets/loaders/mnist.py

```diff
@@ -1,44 +1,99 @@
 import random
 import gzip
+import numpy as np
 from srsly import cloudpickle as pickle
 
 from ..util import unzip, to_categorical, get_file
 from .._registry import register_loader
 
 
-URL = "https://s3.amazonaws.com/img-datasets/mnist.pkl.gz"
+MNIST_URL = "https://s3.amazonaws.com/img-datasets/mnist.pkl.gz"
+
+FA_TR_IMG_URL = "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/train-images-idx3-ubyte.gz"
+FA_TR_LBL_URL = "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/train-labels-idx1-ubyte.gz"
+FA_TE_IMG_URL = "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/t10k-images-idx3-ubyte.gz"
+FA_TE_LBL_URL = "http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/t10k-labels-idx1-ubyte.gz"
+
+KU_TR_IMG_URL = "http://codh.rois.ac.jp/kmnist/dataset/kmnist/kmnist-train-imgs.npz"
+KU_TR_LBL_URL = "http://codh.rois.ac.jp/kmnist/dataset/kmnist/kmnist-train-labels.npz"
+KU_TE_IMG_URL = "http://codh.rois.ac.jp/kmnist/dataset/kmnist/kmnist-test-imgs.npz"
+KU_TE_LBL_URL = "http://codh.rois.ac.jp/kmnist/dataset/kmnist/kmnist-test-labels.npz"
```

Comment:
I realise these are long names, and I agree with abbreviating them, but let's maybe put `TRAIN` and `TEST` in full instead of just `TR` and `TE` to improve readability somewhat